projects/core/src/auth/user-auth/facade/auth.service.ts

```diff
@@ -85,13 +87,20 @@ export class AuthService {
   /**
    * Revokes tokens and clears state for logged user (tokens, userId).
    * To perform logout it is best to use `logout` method. Use this method with caution.
+   * @param showGlobalMsg show a successful global message upon sign out.
    */
-  coreLogout(): Promise<void> {
+  coreLogout(showGlobalMsg?: boolean): Promise<void> {
```

Comment:
```suggestion
  coreLogout(showGlobalMsg = true): Promise<void> {
```

Can we provide `true` as a default?

And in this case can you pass `false` value where we call it in `AuthHttpHeaderService` and in `UpdateEmailComponentService`.

For these 2 cases, it doesn't make sense to show this message.